crawler/src/main/java/com/movie/recommender/crawler/Films.java

```diff
@@ -0,0 +1,85 @@
+package com.movie.recommender.crawler;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+import java.io.IOException;
+
+public class Films {
+    private static final String TOKEN = "Bearer eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiIzMTI3NGFmYTRlNTUyMjRjYzRlN2Q0NmNlMTNkOTZjOSIsInN1YiI6IjVkNmZhMWZmNzdjMDFmMDAxMDU5NzQ4OSIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.lbpgyXlOXwrbY0mUmP-zQpNAMCw_h-oaudAJB6Cn5c8";
+    private static String popular_movies_url = "https://api.themoviedb.org/3/discover/movie?sort_by=popularity.desc&page=";
+    private final int pages;
+    public JsonArray film_results_array = new JsonArray();
+    public JsonObject film_results;
+    public JsonArray genres;
+    public Films(int pages) {
+        this.pages = pages;
+        fetchDataForPages();
+        fetchGenres();
+    }
+
+    private JsonArray fetchData(int page,String question_url) {
```

Comment:
use proper method names for consistency:

- getNowPlaying
- getMovieDetails
- getMovieCredits
- getMovieKeywords